src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -907,6 +907,76 @@ public override IOperation VisitWhileLoop(IWhileLoopOperation operation, int? ca
             return null;
         }
 
+        public override IOperation VisitVariableDeclarationGroup(IVariableDeclarationGroupOperation operation, int? captureIdForResult)
+        {
+            // We erase variable declarations from the control flow graph, as variable lifetime information is contained in a parallel data structure.
+            foreach (var declaration in operation.Declarations)
+            {
+                HandleVariableDeclaration(declaration, captureIdForResult);
+            }
+
+            return null;
+        }
+
+        private void HandleVariableDeclaration(IVariableDeclarationOperation operation, int? captureIdForResult)
+        {
+            foreach (IVariableDeclaratorOperation declarator in operation.Declarators)
+            {
+                ILocalSymbol localSymbol = declarator.Symbol;
+
+                IOperation initializer = null;
+                SyntaxNode assignmentSyntax = null;
+                if (declarator.Initializer != null)
+                {
+                    initializer = Visit(declarator.Initializer.Value, captureIdForResult);
```

Comment:
>captureIdForResult [](start = 70, length = 18)

We should not be propagating ```captureIdForResult``` here. We are not producing a value here, the parameter should be removed from the calls and the enclosing function.

Reply:
Removed.

---
In reply to: [167019822](https://github.com/dotnet/roslyn/pull/24692#discussion_r167019822) [](ancestors = 167019822)